src/Provider/Nominatim/Model/NominatimAddress.php

```diff
@@ -73,24 +73,34 @@ public function withAttribution(string $attribution = null): self
     /**
      * @return null|string
      */
+    public function getCategory()
+    {
+        return $this->category;
+    }
+
     public function getClass()
     {
-        return $this->class;
+        return $this->getCategory();
     }
 
     /**
-     * @param null|string $class
+     * @param null|string $category
      *
      * @return NominatimAddress
      */
-    public function withClass(string $class = null): self
+    public function withCategory(string $category = null): self
     {
         $new = clone $this;
-        $new->class = $class;
+        $new->category = $category;
 
         return $new;
     }
 
+    public function withClass(string $category = null): self
```

Comment:
Why do we need this? Dont we rename `class` to `category`?

Reply:
Oh, Yes of course. 

The diff was weird. I thought you added a new "withClass" function